### PR TITLE
CD FAQs

### PR DESCRIPTION
**Pull request by austinthao5**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 1]
```diff
@@ -6454,3 +6454,60 @@ Please read more on this in the following [Doumentation](https://developer.harne
 
 Yes, Harness provides the expression `<+pipeline.executionUrl>` to fetch the current variable pipeline execution Url
 Please read more on pipeline expressions in the following [Documentation](https://developer.harness.io/docs/platform/variables-and-expressions/harness-variables/#pipelineexecutionurl)
+
+#### Why am I getting an Invalid Repository Error on my Pipeline Trigger?
+
+If you've recently migrated your Pipeline and Input Set(s) from Inline to Remote, you may encounter this error. To fix this, the Trigger needs to reference a Remote Input Set. If using Remote Input Sets and Remote Pipelines, it is required for Triggers to reference a Remote Input Set.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6460]:
```suggestion
If you've recently migrated your Pipeline and Input Set(s) from Inline to Remote, you may encounter this error. To fix this, the Trigger needs to reference a Remote Input Set.  It is required for Triggers to reference a Remote Input Set when using Remote Input Sets and Remote Pipelines.
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 2]
```diff
@@ -6454,3 +6454,60 @@ Please read more on this in the following [Doumentation](https://developer.harne
...
+
+#### Why am I getting an error that my trigger has empty or missing pipelineBranchName?
+
+```
+Failed while requesting Pipeline Execution through Trigger: Unable to continue trigger execution. Pipeline with identifier: $PIPELINE_ID, with org: $ORG, with ProjectId: $PROJ, For Trigger: $TRIGGER has missing or empty pipelineBranchName in trigger's yaml.
+```
+
+If you've recently migrated your Pipeline and Input Set(s) from Inline to Remote, you may encounter this error. To fix this, the Trigger needs to reference a Remote Input Set. If using Remote Input Sets and Remote Pipelines, it is required for Triggers to reference a Remote Input Set.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6468]:
```suggestion
If you've recently migrated your Pipeline and Input Set(s) from Inline to Remote, you may encounter this error. To fix this, the Trigger needs to reference a Remote Input Set. It is required for Triggers to reference a Remote Input Set when using Remote Input Sets and Remote Pipelines.
```

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 3]
```diff
@@ -6454,3 +6454,60 @@ Please read more on this in the following [Doumentation](https://developer.harne
...
+
+
+#### What's the difference between matchType all and any?
+
+If using a Filtered List to deploy to multiple environments, you can dynamically set which environments to deploy to using tags. The `matchType` field is used to define the operator for the tags list.
+
+All - Only deploy to environments matching all the tags.
+
+Any - Deploy to environments matching any of the tags.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6477]: Fantastic explanation

File: kb/continuous-delivery/continuous-delivery-faqs.md [hunk 4]
```diff
@@ -6454,3 +6454,60 @@ Please read more on this in the following [Doumentation](https://developer.harne
...
+
+
+#### Why is my pipeline timing out even though my step hasn't reached the timeout yet?
+
+If your pipeline is timing out before your step has a chance to hit the timeout threshold, it's likely that the pipeline itself has a timeout that has already been reached.
```
Comment (SushrutHarness) [inline kb/continuous-delivery/continuous-delivery-faqs.md:6482]: Can you quickly tell me (the customer) where to find this timeout setting for the pipeline?

**Review by SushrutHarness — CHANGES_REQUESTED** [review]
Great FAQ writing! I left a few nitpicks and few suggestions, take a look and check if they're still right technically.

File: kb/continuous-delivery/continuous-delivery-faqs.md (hunk 4 above)
Comment (austinthao5) [reply kb/continuous-delivery/continuous-delivery-faqs.md:6482]: Sure, it's in Advanced Settings > Pipeline Timeout Settings

**Review by SushrutHarness — DISMISSED** [review]
Consider adding the setting instructions to FAQ. OTherwise looks good

**Review by SushrutHarness — APPROVED** [review]

**Pull request merged**